sdk/data/azcosmos/cosmos_item_request_patch_options.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+// PatchOperationType defines supported values for operation types in Patch Document.
+type PatchOperationType string
+
+const (
+	// Represents a patch operation Add.
+	PatchOperationTypeAdd PatchOperationType = "add"
+	// Represents a patch operation Replace.
+	PatchOperationTypeReplace PatchOperationType = "replace"
+	// Represents a patch operation Remove.
+	PatchOperationTypeRemove PatchOperationType = "remove"
+	// Represents a patch operation Set.
+	PatchOperationTypeSet PatchOperationType = "set"
+	// Represents a patch operation Increment.
+	PatchOperationTypeIncrement PatchOperationType = "incr"
+)
+
+// PatchOperation represents individual patch operation.
+type PatchOperation struct {
+	Op    PatchOperationType `json:"op"`
+	Path  string             `json:"path"`
+	Value interface{}        `json:"value,omitempty"`
+}
+
+// PatchOptions represents the patch request.
+type PatchOptions struct {
```

Comment:
[From API review] Rename to `PatchOperations` make the fields private, they will be set through the helper functions